boards/DCM/Src/Io/Io_LSM6DS33.c

```diff
@@ -0,0 +1,214 @@
+#include "Io_LSM6DS33.h"
+#include "App_Imu.h"
+
+// The I2c slave device addresses to use when reading/writing to/from the IMU
+// See table 11 in section 6.1.1 of the LSM6DS33 data sheet for details
+// Register Address Constants
+#define FUNC_CFG_ACCESS 0x01
+#define ORIENT_CFG_G 0x0B
+#define INT1_CTRL 0x0D
+#define INT2_CTRL 0x0E
+#define CTRL1_XL 0x10
+#define CTRL2_G 0x11
+#define CTRL3_C 0x12
+#define CTRL4_C 0x13
+#define CTRL6_C 0x15
+#define CTRL7_C 0x16
+#define CTRL8_XL 0x17
+#define CTRL9_XL 0x18
+#define CTRL10_C 0x19
+#define STATUS_REG 0x1E
+#define DATA_OUT_STRT 0x22
+#define IMU_READ_ADDR 0xd7
+#define IMU_WRITE_ADDR 0xd6
+
+struct ImuData
+{
+    // Acceleration in m/s^2
+    float accel_x;
+    float accel_y;
+    float accel_z;
+
+    // The time (in ms) that this data was received
+    uint32_t received_time_ms;
+};
+
+struct LSM6DS33
+{
+    I2C_HandleTypeDef *imu_i2c_handle;
+};
+
+static struct LSM6DS33 lsm_6ds33;
+
+static struct ImuData most_recently_received_data = { 0 };
```

Comment:
When I suggested we use a struct I was thinking that we put the remaining variables below inside the struct as well

Reply:
ya my bad, didn't look carefully enough for other static vars